hooli_data_eng/assets/dbt_assets.py

```diff
@@ -24,6 +24,7 @@
 )
 from dagster_dbt.asset_decorator import dbt_assets
 from dagster._utils import file_relative_path
+from ..resources import dbt_resource
```

Comment:
I would generally prefer absolute reference paths, 
```
hooli_data_eng.resources` or whatever

Reply:
this code is gone now but 👍  for the future